frontend/components/GamesVisibilityList.tsx

```diff
@@ -27,14 +26,14 @@ const GameVisibilityToggle: React.FC<GameVisibilityToggleProps> = (props) =>
 		<input type="checkbox" checked = {props.isVisible} name={`${props.game.name}: ${props.game.id}`} id={`spoiler #${props.game.id}`}/> {props.game.name}
 	</div>;
 
-const mapStateToProps = (state: AppStore): GameVisibilityListProps => {
+const mapStateToProps = (state: AppStore): GamesVisibilityListProps => {
 	const {games, gamesVisibility} = state;
-	const gameVisibilityToggleMap = new Map<number, GameVisibilityToggleProps>();
+	const gamesVisibilityToggleArray = new Array<GameVisibilityToggleProps>();
```

Comment:
Instead of having `Array` at the end of the name (which makes it into a kind of awkward [systems hungarian](https://en.wikipedia.org/wiki/Hungarian_notation)) just make the base plural - `gameVisibilityToggles`.

Also a subtle semantic difference between `gamesVisibilityToggle`, `gameVisibilityToggle`, `gameVisibilityToggles` and `gamesVisibilityToggles`:

 - The first is **one** toggle for **many** games
 - The second is **one** toggle for **one** games
 - The third is **many** toggles where each one of them is for **one** game
 - The last one is **many** toggles where each one of them is for **many** games

[Naming things is hard](https://martinfowler.com/bliki/TwoHardThings.html)!

Reply:
Which also means you should probably change the name of the components and the file, and remember to change all the usages to have correct puralizations.